pkgs/development/interpreters/clojure/babashka-unwrapped.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
```

Comment:
Can we reorganize this in a subdirectory `babashka`? And then this is the `unwrapped.nix` file while the wrapper is `default.nix`.

Reply:
Or this is the `default.nix` and the other is the `wrapper.nix`. Not sure how is the common way to organize this.

`mpv` is again a good example.

Reply:
Makes sense, I went with `default.nix` for the unwrapped version, looking at mpv and neovim, seems the convention.